### ISSUE TITLE
benchmark deepaudiox on HEAR

### ISSUE THREAD

**Issue by ChrisNick92** [magcil/deepaudio-x#46]
Evaluate the deepaudiox library on the [HEAR](https://hearbenchmark.com/) Benchmark.

**About HEAR**

HEAR is a holistic evaluation pipeline aiming to provide a robust evaluation pipeline for testing the efficiency of audio representations. It consists of a total of nineteen diverse tasks. These tasks encompass multiple audio domains: speech, environmental sound, and music, with tasks that involve short and long time spans. 

16/19 are multiclass - binary classification tasks and the 3/19 are multi-label classification tasks. Read the official paper [here](https://arxiv.org/abs/2203.03022) to familiarize with the tasks, splitting methods, and evaluation protocols.

**Evaluation DeepAudioX on HEAR**

1. In our case we want to evaluate the efficiency (easy-to-use + performance) of DeepAudioX on HEAR. I've downloaded 15/16 datasets of multiclass-binary classification category.

2. I've created an .xlsx file in [here](https://docs.google.com/spreadsheets/d/1TnVFvnq-zIVlbVWg4cRG0Gp61fdEWT1e9ztn_nNTrYM/edit?usp=sharing) to track the accuracy of each run (for different backbones, pooling methods and sample rates).

3. **IMPORTANT**: These results will be used in our paper about to be submitted in JOSS. To this end, we want a **fair** comparison for each combination. To achieve this, please use the **default hyperparameters** mentioned in the .xlsx file. 

You don't need to change anything from the default values of the `Trainer`, only the number of epochs. batch_size, and num workers. You simple initialize:

```
from deepaudiox.loops.trainer import Trainer

trainer = Trainer(train_dset=train_dset, model=model, validation_dset=val_dset, epochs=200, batch_size=256, num_workers=8)
```
